2repository/6.2/ent-rename-for-build-6.2.1/overload.gradle

```diff
@@ -3,7 +3,8 @@ ext {
 	version: [
```

Comment:
Is it on purpose that this one does no longer build now?

Reply:
I just noticed there was a pattern to disable build for old minor versions, which I followed.